app/main.py

```diff
@@ -1,2 +1,90 @@
+from dataclasses import dataclass
+from typing import Hashable, Iterator
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: any
+    hash_value: int
+
+
 class Dictionary:
-    pass
+    INITIAL_CAPACITY = 8
+    THRESHOLD = 2 / 3
```

Comment:
The calculation of the THRESHOLD is incorrect. It should be a float representing the load factor threshold, such as 0.75, not an integer division which results in 0.